website/themes/uppy/layout/index.ejs

```diff
@@ -174,7 +174,7 @@
 <section class="IndexActivity">
   <h2>Activity Feed</h2>
 
-  <div class="on-the-githubs" data-event-source="repos/transloadit/uppy">Loading...</div>
+  <ul class="on-the-githubs" data-event-source="repos/transloadit/uppy">Loading...</ul>
```

Comment:
Certainly an improvement, however I don't think if it's allowed to have a "naked" text node (`Loading...`), <validator.w3.org> does report is as an error. We also lack a fallback for when JS is disabled and the feed will never load. What we could do is:

```suggestion
<noscript class="activity-feed" data-event-source="repos/transloadit/uppy">You need JavaScript to display the activity feed.</noscript>
```

and then update the JS https://github.com/transloadit/uppy/blob/d6a5695e391493c83769af0437ea86c4bcc00527/website/themes/uppy/layout/layout.ejs#L103

```js
          {
            const $el = $('noscript.activity-feed');
            $('<ul class="on-the-githubs">').attr('data-event-source', $el.attr('data-event-source')).replace($el).onthegithubs();
          }
```